proto/ibc/tendermint/tendermint.proto

```diff
@@ -68,6 +68,26 @@ message ConsensusState {
   ];
 }
 
+// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
```

Comment:
```suggestion
// Evidence is a wrapper over two conflicting Headers
```

Reply:
thanks for fixing this, I just copied the existing godoc